src/lib/helpers.ts

```diff
@@ -13,7 +13,7 @@ export function uuid() {
   })
 }
 
-export const isBrowser = () => typeof window !== 'undefined'
+export const isBrowser = () => typeof document !== 'undefined'
```

Comment:
hmm i'm not too familiar with the `document` and `window` objects in react native (RN), can you please explain or drop some relevant links to how they work in RN? 

also, have you tested that this works on a browser in a non-RN setting?